index.html

```diff
@@ -56,6 +57,149 @@ <h3 class="connect">LET'S CONNECT</h3>
             </a>
           </li>
         </ul>
+        <div class="bottomspace"></div>
+      </section>
+
+      <!-- Works Section -->
+      <section class="works">
+
+        <!-- card 1 -->
+        <div class="card card1">
+          <img class="card-cover" src="src/images/card-cover1.png" alt="">
```

Comment:
- [ ] Your work section is looking great already, I just have one suggestion here. I notice your margin-top is a bit much compared with the figma design. Please minimize the space between the heading and the work section to make your design even better. 

![Screenshot 2023-02-17 200924](https://user-images.githubusercontent.com/69077061/219762438-d4ffae49-85d2-422a-8bd3-8d084236519e.png)